ciao-controller/internal/datastore/datastore.go

```diff
@@ -832,7 +783,9 @@ func (ds *Datastore) GetAllInstancesByNode(nodeID string) ([]*types.Instance, er
 	n, ok := ds.nodes[nodeID]
 	if ok {
 		for _, val := range n.instances {
-			instances = append(instances, val)
+			if val.CNCI == false {
```

Comment:
we always have, so if we don't exclude we've changed behavior. The caller of this is an api which is meant to return all the "servers" on a node. This is defined by open stack, and this doesn't include CNCI, which is supposed to be sort of invisible.